Antidote/CallsManager.m

```diff
@@ -271,6 +275,43 @@ - (void)ringingCallDeclineButtonPressed:(RingingCallViewController *)controller
                             error:nil];
 }
 
+#pragma mark - ActiveCallViewControllerDataSource
+
+- (NSInteger)activeCallControllerNumberOfPausedCalls:(ActiveCallViewController *)controller
+{
+    return [[self.allPausedCallsController fetchedObjects] count];
+}
+
+- (NSString *)activeCallController:(ActiveCallViewController *)controller pausedCallerNicknameForCallAtIndex:(NSIndexPath *)indexPath
+{
+    OCTCall *call = [self.allPausedCallsController objectAtIndexPath:indexPath];
+
+    OCTFriend *friend = [call.chat.friends firstObject];
+
+    return friend.nickname;
+}
+
+- (NSTimeInterval)activeCallController:(ActiveCallViewController *)controller pauseTimeDurationForCallAtIndex:(NSIndexPath *)indexPath
+{
+    OCTCall *call = [self.allPausedCallsController objectAtIndexPath:indexPath];
+
+    return call.callDuration;
```

Comment:
I don't think that timer should go to objcTox. So as I see it:
- add `onHoldStartDate` to objcTox (or `onHoldStartInterval` for saving precision)
- in ActiveCallVC, when table is visible start timer, that will tick every second
- when table isn't visible, remove timer.